### ISSUE TITLE
Custom code fence per code block

### ISSUE THREAD

**Issue by efaulhaber** [fredrikekre/Literate.jl#264]
According to the docs,
> - The default code fence will change from
>   ````
>   ```julia
>   # code
>   ```
>   ````
>
>   to Documenters @example blocks:
>
>   ````
>   ```@examples $(name)
>   # code
>   ```
>   ````

And in the config, I noticed that this can be changed with `codefence`.
However, I would like to use a different fence only a single code block, which should read
````
```@cast
# code
```
````
to use [Asciicast.jl](https://github.com/ericphanson/Asciicast.jl). See the docs [here](https://ericphanson.github.io/Asciicast.jl/stable/documenter_usage/). Of course, I don't want all blocks to generate Asciinema output.

Is this currently possible?

**Comment by fredrikekre** [comment]
You can do
````julia
# ```@cast
# # code
# ```
````

**Comment by efaulhaber** [comment]
Cool! But then I would have to do something like
````
# ```@cast
# my_code()
# ```
my_code() #!md
````
so that `my_code()` is
1. run when I include the file,
2. included in the notebook and
3. not executed twice in Documenter.jl.

Right?

**Comment by fredrikekre** [comment]
Yes. If you have more code that depends on this I think you need to evaluate it twice anyway (otherwise the variables won't be in the same anonymous module).
